src/list/popup/components/item-list-panel.js

```diff
@@ -35,16 +35,16 @@ class PopupItemList extends React.Component {
   render() {
     const {items, ...props} = this.props;
     const {selected} = this.state;
-    const verbose = items.length <= MAX_VERBOSE_ITEMS;
 
     return (
-      <ItemList {...props} items={items} verbose={verbose} selected={selected}
+        <ItemList {...props} panel={true} items={items} verbose={false} selected={selected}
```

Comment:
This causes breaking in `test/unit/list/popup/containers/all-items-panel-test.js`.  We'd have to look back over the designs, but I think it's safe enough to remove `verbose` altogether.

Otherwise, [this check](https://github.com/mozilla-lockbox/lockbox-addon/blob/c36df2c25d65161c72c3eb5021a153879040d876/test/unit/list/popup/containers/all-items-panel-test.js#L100) should be removed.